src/services/BooksService.js

```diff
@@ -0,0 +1,15 @@
+import http from '../http-common';
+
+const getAll = (appId) => http.get(`/apps/${appId}/books`);
+
+const create = (appId, data) => http.post(`/apps/${appId}/books`, data);
+
+const remove = (appId, id) => http.delete(`/apps/${appId}/books/${id}`);
```

Comment:
- [ ] Please use `fetch` or `Axios` for making API calls as the project requirement mentioned not using the old HTTP request method.

![image](https://user-images.githubusercontent.com/68796232/179252168-c9076834-b17c-491a-8d19-ec98fc4a1a53.png)

Reply:
I actually used Axios. If you have taken time to check the http-common.js file located in ths src file, you would have seen it